README.md

```diff
@@ -56,8 +56,8 @@ In your mix file, include the additional dependency
 #mix.exs
 def deps() do
   [
-    {:divo, "~> 1.3.1", only: [:dev, :integration]},
-    {:divo_redis, "~> 0.1.0", only: [:dev, :integration]}
+    {:divo, "~> 2.0.0", only: [:dev, :integration]},
+    {:divo_redis, "~> 1.0.0", only: [:dev, :integration]}
```

Comment:
Does divo_redis 1.0.0 exist? https://hexdocs.pm/divo_redis/readme.html

Reply:
I'm placing this here for when I get Divo_redis updated, as it is next, so I don't have to come back and update the readme here